### DISCUSSION TITLE
Make tablet support more obvious.

### DISCUSSION THREAD

**Discussion by LeNitrous** [ppy/osu#23772] [Ideas]
In my time during CONQuest Festival 2023 we had several PCs running osu! lazer. As per observations, I have noticed that several users have opened OpenTabletDriver to configure their settings after plugging in their graphic tablets. I had to approach these users and explain to them that it is no longer needed as it is a built-in feature in the client itself. They were surprised that it was a thing.

No where in the UX hints of native tablet support within the game itself. It would be nice if there was a hint or notification of any kind that this is possible.

**Comment by peppy** [comment]
https://github.com/ppy/osu/issues/18884 might cover this. it could be nice to have notifications showing connected tablets though
